### PR TITLE
added pha-202+

### PR DESCRIPTION
**Pull request by mattcoates**
https://www.minicircuits.com/WebStore/dashboard.html?model=PHA-202%2B

### CODE REVIEW

File: scripts/build_mod_ic.py [hunk 1]
```diff
@@ -442,7 +442,7 @@
         "ep_vias": (0.4, 0.6, 1.9),
         "chip_shape": (7.1, 7.1),
         "pin_shape": (-0.5, 0.3),
-        "model": {
+       "model": {
```
Comment (adamgreig) [inline scripts/build_mod_ic.py]: You seem to have accidentally deleted a space from this line?

File: scripts/build_mod_ic.py [hunk 2]
```diff
@@ -502,7 +502,20 @@
...
+    # For PHA-202+
+    "DL1636": {
+        "rows": 2,
+        "pins": 8,
+        "pin_pitch": 1.27,
+        "row_pitch": 5.51,
+        "pad_shape": (1.02, 0.51),
+        "ep_shape": (3.94, 4.11),
+        "chip_shape": (6, 4.90),
+        "pin_shape": (-0.6, 0.42),
```
Comment (adamgreig) [inline scripts/build_mod_ic.py]: You can also add the EP vias the datasheet recommends: `"ep_vias": (0.5, 1.0, 1.0)` or similar, adjust to fit: (drill, size, gap).

**Comment by mattcoates** [comment]
It seems when I made those changes I also added a footprint for this inductor:
https://www.coilcraft.com/pdfs/wa8514.pdf

File: scripts/build_mod_ic.py (hunk 2 above)
Comment (adamgreig) [reply scripts/build_mod_ic.py]: Using a gap of 0.5 has given three rows of vias but the footprint datasheet has 2; using 0.8 for the gap instead gives you two rows.

File: scripts/build_mod_ic.py [hunk 3]
```diff
@@ -502,7 +502,21 @@
...
+        "ep_vias": (0.4, 0.8, 0.5),
+    },
+
```
Comment (adamgreig) [inline scripts/build_mod_ic.py]: Also please delete these trailing spaces before and after this block.

**Pull request merged**